File: spring/src/main/java/dev/stinner/scoutventure/domain/models/NamiMember.java
package dev.stinner.scoutventure.domain.models;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDate;
import java.util.Set;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class NamiMember {
    private Long memberId;
    private String firstname;
    private String lastname;

    private LocalDate dateOfBirth;

    private Rank rank;
    private Gender gender;

    private Set<User> userAssignments;
}

File: asp-net/ScoutVenture/CoreContracts/NamiMember/NamiMember.cs
namespace ScoutVenture.CoreContracts.NamiMember
{
    public class NamiMember
    {
        long MemberId { get; set; }
        string? FirstName { get; set; }
        string? LastName { get; set; }
        DateTimeOffset DateOfBirth { get; set; }

    }
}
